### ISSUE TITLE
[FBref] match-level opponent data not available for read_team_match_stats

### ISSUE THREAD

**Issue by mhd0528** [probberechts/soccerdata#370]
Hi, is getting opponent team info from `read_team_match_stats `still supported? I got the following error. Any suggestions how to fix this? Thanks in advance!
```
File "/home/anaconda3/envs/soccerdata/lib/python3.10/site-packages/soccerdata/fbref.py", line 432, in read_team_match_stats
    (html_table,) = tree.xpath(f"//table[@id='matchlogs_{opp_type}']")
ValueError: not enough values to unpack (expected 1, got 0)
```

**Comment by probberechts** [comment]
They are simply not available for the "schedule" stat type

**Comment by mhd0528** [comment]
Ok thanks! I'll look into other types of stats first then.

**Comment by mhd0528** [comment]
Okay so I tried to disable the local cache and with other stat types other than schedule, but it still gives me a scraping error like below:
```
Error while scraping                                                         _common.py:351
                             [](https://fbref.com/en/squads/18bb7c10/matchlogs/all_comps/shooting.)
```
And I checked this url, it is not valid in FBref. The closest valid link I can find is: https://fbref.com/en/squads/18bb7c10/2023-2024/matchlogs/all_comps/shooting/Arsenal-Match-Logs-All-Competitions. Is it possible that there is some error in the url generation? Sorry for bothering you recently, I'm trying to figure out what's the error on my end, but can't solve it...
